### DISCUSSION TITLE
Auto reload like alacritty

### DISCUSSION THREAD

**Discussion by valexand-ulb** [kovidgoyal/kitty#5416] [Q&A] [answered]
I've created my first bspwm theme compatible with alacritty, kitty, rofi, polybar and thunar and i was wondering if there were a settings that i can add in the config file to make kitty reload (like "live_config_reload" of alacritty) ?

**Comment by kovidgoyal** [comment, marked as answer]
There is no auto relaod you press kitty_mod+f5 to reload or send kitty
processes the SIGUSR1 signal

**Comment by megalithic** [comment]
Here's what I do to do this on save in neovim:

https://github.com/megalithic/dotfiles/blob/328f646263aceeab029561e0b6586a9ac310b367/config/nvim/plugin/autocmds.lua#L154-L164
